doc/chips/11.rst

```diff
@@ -0,0 +1,183 @@
+Alternative Initializer Features
+================================
+
+Status:
+  Rejected
+
+Authors:
+  Kyle Brady, Lydia Duncan, Michael Ferguson, Mike Noakes, Vassily Litvinov (for Syntax 2);
+  Vassily Litvinov (for Child Fields and Initialization Completion)
+
+Abstract
+--------
+
+This document contains the features we discussed as part of the
```

Comment:
"the features" -> "some features".  One implies complete coverage, which I believe you (reasonably) have not attempted